src/models/server.ts

```diff
@@ -9,4 +10,5 @@ export interface ServerInterface extends BaseModel, DescriptionMixinInterface, B
   protocolVersion(): string;
   hasProtocolVersion(): boolean;
   variables(): ServerVariablesInterface
+  securitySchemes(): SecuritySchemesInterface
```

Comment:
```suggestion
```

We have bug in `parser-api`. That field should name as `security` and should return [`Security Requirements Object`](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#securityRequirementObject) that we don't have described in `parser-api`.

it will be easier to remove from this PR that field.